tests/functional/pv/pvc_snapshot/test_rbd_cbt_metadata.py

```
@@ -0,0 +1,423 @@
+        write_data_to_pvc(
+            writer_pod,
+            volume_mode,
+            size_mb=5,
+            filename="file2.bin",
+            offset_mb=100,
+        )
+
+        log.test_step("Take snap-2")
+        snap_2 = self._take_snapshot(app_pvc, "cbt-incr-2")
+
+        log.test_step("Run the CBT lister in delta mode " "(snap-2 vs snap-1)")
```

Ruff (0.16.1)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)